### PR TITLE
Adapt for dbms

### PR DESCRIPTION
**Pull request by olegrok**
```
pytest -p no:seleniumbase
=================================================================== test session starts ====================================================================
platform linux -- Python 3.10.6, pytest-7.2.1, pluggy-1.0.0 -- /usr/bin/python3
cachedir: .pytest_cache
metadata: {'Python': '3.10.6', 'Platform': 'Linux-5.15.79.1-microsoft-standard-WSL2-x86_64-with-glibc2.35', 'Packages': {'pytest': '7.2.1', 'pluggy': '1.0.0'}, 'Plugins': {'html': '2.0.1', 'typeguard': '2.13.3', 'rerunfailures': '11.0', 'metadata': '2.0.4', 'ordering': '0.6', 'xdist': '3.1.0', 'forked': '1.4.0'}}
rootdir: /home/olegpc/Projects/dbms-python, configfile: pyproject.toml, testpaths: tests
plugins: html-2.0.1, typeguard-2.13.3, rerunfailures-11.0, metadata-2.0.4, ordering-0.6, xdist-3.1.0, forked-1.4.0
collected 150 items

tests/test_analyzer.py::test_analyzer_management[db0-bad_db0] PASSED
tests/test_aql.py::test_aql_attributes[db0] PASSED
tests/test_aql.py::test_aql_query_management[db0-bad_db0] PASSED
tests/test_aql.py::test_aql_function_management[db0-bad_db0] PASSED
tests/test_aql.py::test_aql_cache_management[db0-bad_db0] PASSED
tests/test_async.py::test_async_wrapper_attributes[db0] PASSED
tests/test_async.py::test_async_execute_without_result[db0] PASSED
tests/test_async.py::test_async_execute_error_in_result[db0] PASSED
tests/test_async.py::test_async_get_job_status[db0-bad_db0] PASSED
tests/test_async.py::test_async_get_job_result[db0-bad_db0] PASSED
tests/test_async.py::test_async_cancel_job[db0-bad_db0] PASSED
tests/test_async.py::test_async_clear_job[db0-bad_db0] PASSED
tests/test_async.py::test_async_execute_errors[bad_db0] PASSED
tests/test_async.py::test_async_clear_jobs[db0-bad_db0] PASSED
tests/test_async.py::test_async_list_jobs[db0] PASSED
tests/test_auth.py::test_auth_invalid_method PASSED
tests/test_auth.py::test_auth_basic PASSED
tests/test_auth.py::test_auth_jwt PASSED
tests/test_auth.py::test_auth_superuser_token PASSED
tests/test_auth.py::test_auth_jwt_expiry PASSED
tests/test_backup.py::test_backup_management[bad_db0] SKIPPED (Only for DbmsDB enterprise edition)
tests/test_batch.py::test_batch_wrapper_attributes[db0] PASSED
tests/test_batch.py::test_batch_execute_without_result[db0] PASSED
tests/test_batch.py::test_batch_execute_with_result[db0] PASSED
tests/test_batch.py::test_batch_empty_commit[db0] PASSED
tests/test_batch.py::test_batch_double_commit[db0] PASSED
tests/test_batch.py::test_batch_action_after_commit[db0] PASSED
tests/test_batch.py::test_batch_execute_error[db0-bad_db0] PASSED
tests/test_batch.py::test_batch_job_result_not_ready[db0] PASSED
tests/test_batch.py::test_batch_bad_state[db0] PASSED
tests/test_client.py::test_client_attributes PASSED
tests/test_client.py::test_client_good_connection[db0] PASSED
tests/test_client.py::test_client_bad_connection[db0] PASSED
tests/test_client.py::test_client_custom_http_client[db0] PASSED
tests/test_client.py::test_client_override_validate PASSED
tests/test_cluster.py::test_cluster_server_id[bad_db0] SKIPPED (Only tested in a cluster setup)
tests/test_cluster.py::test_cluster_server_role[bad_db0] SKIPPED (Only tested in a cluster setup)
tests/test_cluster.py::test_cluster_health[bad_db0] SKIPPED (Only tested in a cluster setup)
tests/test_cluster.py::test_cluster_server_version[bad_db0] SKIPPED (Only tested in a cluster setup)
tests/test_cluster.py::test_cluster_server_engine[bad_db0] SKIPPED (Only tested in a cluster setup)
tests/test_cluster.py::test_cluster_server_statistics[bad_db0] SKIPPED (Only tested in a cluster setup)
tests/test_cluster.py::test_cluster_toggle_maintenance_mode[bad_db0] SKIPPED (Only tested in a cluster setup)
tests/test_cluster.py::test_cluster_endpoints[db0-bad_db0] SKIPPED (Only tested in a cluster setup)
tests/test_cluster.py::test_cluster_server_count[db0-bad_db0] SKIPPED (Only tested in a cluster setup)
tests/test_cursor.py::test_cursor_from_execute_query[db0] PASSED
tests/test_cursor.py::test_cursor_write_query[db0] PASSED
tests/test_cursor.py::test_cursor_invalid_id[db0] PASSED
tests/test_cursor.py::test_cursor_premature_close[db0] PASSED
tests/test_cursor.py::test_cursor_context_manager[db0] PASSED
tests/test_cursor.py::test_cursor_manual_fetch_and_pop[db0] PASSED
tests/test_cursor.py::test_cursor_no_count[db0] PASSED
tests/test_database.py::test_database_attributes[db0] PASSED
tests/test_database.py::test_database_misc_methods[db0-bad_db0] PASSED
tests/test_database.py::test_database_management[db0-bad_db0] PASSED
tests/test_document.py::test_document_insert[db0] PASSED
tests/test_document.py::test_document_insert_many[db0-bad_db0] PASSED
tests/test_document.py::test_document_update[db0] PASSED
tests/test_document.py::test_document_update_many[db0-bad_db0] PASSED
tests/test_document.py::test_document_update_match[db0-bad_db0] PASSED
tests/test_document.py::test_document_replace[db0] PASSED
tests/test_document.py::test_document_replace_many[db0-bad_db0] PASSED
tests/test_document.py::test_document_replace_match[db0-bad_db0] PASSED
tests/test_document.py::test_document_delete[db0] PASSED
tests/test_document.py::test_document_delete_many[db0-bad_db0] PASSED
tests/test_document.py::test_document_delete_match[db0-bad_db0] PASSED
tests/test_document.py::test_document_count[db0-bad_db0] PASSED
tests/test_document.py::test_document_find[db0-bad_db0] PASSED
tests/test_document.py::test_document_find_near[db0-bad_db0] PASSED
tests/test_document.py::test_document_find_in_range[db0-bad_db0] PASSED
tests/test_document.py::test_document_find_in_radius[db0-bad_db0] PASSED
tests/test_document.py::test_document_find_in_box[db0-bad_db0] PASSED
tests/test_document.py::test_document_find_by_text[db0] PASSED
tests/test_document.py::test_document_has[db0-bad_db0] PASSED
tests/test_document.py::test_document_get[db0-bad_db0] PASSED
tests/test_document.py::test_document_get_many[db0-bad_db0] PASSED
tests/test_document.py::test_document_all[db0-bad_db0] PASSED
tests/test_document.py::test_document_ids[db0-bad_db0] PASSED
tests/test_document.py::test_document_keys[db0-bad_db0] PASSED
tests/test_document.py::test_document_random[db0-bad_db0] PASSED
tests/test_document.py::test_document_import_bulk[db0-bad_db0] PASSED
tests/test_document.py::test_document_management_via_db[db0] PASSED
tests/test_exception.py::test_server_error[db0] PASSED
tests/test_exception.py::test_client_error[db0] PASSED
tests/test_foxx.py::test_foxx_attributes[db0] PASSED
tests/test_foxx.py::test_foxx_service_management_json[db0-bad_db0] SKIPPED (FIXME: Is not adapted for DBMS)
tests/test_foxx.py::test_foxx_service_management_file[db0] SKIPPED (FIXME: Is not adapted for DBMS)
tests/test_foxx.py::test_foxx_config_management[db0] SKIPPED (FIXME: Is not adapted for DBMS)
tests/test_foxx.py::test_foxx_dependency_management[db0] SKIPPED (FIXME: Is not adapted for DBMS)
tests/test_foxx.py::test_foxx_development_toggle[db0] SKIPPED (FIXME: Is not adapted for DBMS)
tests/test_foxx.py::test_foxx_misc_functions[db0-bad_db0] SKIPPED (FIXME: Is not adapted for DBMS)
tests/test_graph.py::test_graph_properties[db0-bad_db0] PASSED
tests/test_graph.py::test_graph_management[db0-bad_db0] PASSED
tests/test_graph.py::test_vertex_relation_management[db0-bad_db0] PASSED
tests/test_graph.py::test_edge_definition_management[db0-bad_db0] PASSED
tests/test_graph.py::test_create_graph_with_edge_definition[db0] PASSED
tests/test_graph.py::test_vertex_management[db0-bad_db0] PASSED
tests/test_graph.py::test_vertex_management_via_graph[db0] PASSED
tests/test_graph.py::test_edge_management[db0-bad_db0] PASSED
tests/test_graph.py::test_vertex_edges[db0-bad_db0] PASSED
tests/test_graph.py::test_edge_management_via_graph[db0] PASSED
tests/test_graph.py::test_traverse[db0] PASSED
tests/test_index.py::test_list_indexes[db0-bad_db0] PASSED
tests/test_index.py::test_add_hash_index[db0] PASSED
tests/test_index.py::test_add_skiplist_index[db0] PASSED
tests/test_index.py::test_add_geo_index[db0] PASSED
tests/test_index.py::test_add_fulltext_index[db0] PASSED
tests/test_index.py::test_add_persistent_index[db0] SKIPPED (FIXME: Is not adapted for DBMS)
tests/test_index.py::test_add_ttl_index[db0] PASSED
tests/test_index.py::test_delete_index[db0-bad_db0] PASSED
tests/test_index.py::test_load_indexes[db0-bad_db0] PASSED
tests/test_permission.py::test_permission_management[bad_db0] PASSED
tests/test_pregel.py::test_pregel_attributes[db0] PASSED
tests/test_pregel.py::test_pregel_management[db0] PASSED
tests/test_relation.py::test_relation_attributes[db0] PASSED
tests/test_relation.py::test_relation_misc_methods[db0-bad_db0] PASSED
tests/test_relation.py::test_relation_management[db0-bad_db0] PASSED
tests/test_replication.py::test_replication_dump_methods[db0-bad_db0] PASSED
tests/test_replication.py::test_replication_inventory[bad_db0] PASSED
tests/test_replication.py::test_replication_logger_state[bad_db0] PASSED
tests/test_replication.py::test_replication_first_tick[bad_db0] PASSED
tests/test_replication.py::test_replication_applier[bad_db0] SKIPPED (FIXME: Is not adapted for DBMS)
tests/test_replication.py::test_replication_make_slave[bad_db0] SKIPPED (Only tested for replication)
tests/test_replication.py::test_replication_cluster_inventory[bad_db0] PASSED
tests/test_replication.py::test_replication_server_id[bad_db0] PASSED
tests/test_replication.py::test_replication_synchronize[bad_db0] SKIPPED (Only tested for replication)
tests/test_request.py::test_request_no_data PASSED
tests/test_request.py::test_request_string_data PASSED
tests/test_request.py::test_request_json_data PASSED
tests/test_request.py::test_request_transaction_data PASSED
tests/test_resolver.py::test_bad_resolver PASSED
tests/test_resolver.py::test_resolver_single_host PASSED
tests/test_resolver.py::test_resolver_random_host PASSED
tests/test_resolver.py::test_resolver_round_robin PASSED
tests/test_response.py::test_response[db0] PASSED
tests/test_task.py::test_task_management[db0-bad_db0] PASSED
tests/test_transaction.py::test_transaction_execute_raw[db0] PASSED
tests/test_transaction.py::test_transaction_init[db0-bad_db0] PASSED
tests/test_transaction.py::test_transaction_status[db0] PASSED
tests/test_transaction.py::test_transaction_commit[db0] PASSED
tests/test_transaction.py::test_transaction_abort[db0] PASSED
tests/test_transaction.py::test_transaction_graph[db0] PASSED
tests/test_user.py::test_user_management[bad_db0] PASSED
tests/test_user.py::test_user_change_password PASSED
tests/test_user.py::test_user_create_with_new_database PASSED
tests/test_view.py::test_view_management[db0-bad_db0] PASSED
tests/test_view.py::test_dbmssearch_view_management[db0-bad_db0] PASSED
tests/test_wal.py::test_wal_misc_methods[bad_db0] SKIPPED (WAL not implemented)
tests/test_wal.py::test_wal_tick_ranges[bad_db0] PASSED
tests/test_wal.py::test_wal_last_tick[bad_db0] PASSED
tests/test_wal.py::test_wal_tail[bad_db0] PASSED

============================================================= 129 passed, 21 skipped in 28.44s =============================================================
```

Part of #1, #2

### CODE REVIEW

**Comment by knazarov** [comment]
I don't see a reason to block this PR. Unless anyone would have any objections in a day or so, let's merge it.

**Review by knazarov — APPROVED** [review]

File: docs/aql.rst [hunk 1]
```diff
@@ -4,7 +4,7 @@ AQL
 **DbmsDB Query Language (AQL)** is used to read and write data. It is similar
```
Comment (kemm) [inline docs/aql.rst]: SQL?

File: docs/indexes.rst [hunk 2]
```diff
@@ -49,7 +49,7 @@ on fields ``_from`` and ``_to``. For more information on indexes, refer to
     # Indexes may be added with a name that can be referred to in AQL queries.
```
Comment (kemm) [inline docs/indexes.rst]: SQL?

File: docs/wal.rst [hunk 3]
```diff
@@ -37,7 +37,7 @@ WAL operations can only be performed by admin users via ``_system`` database.
     # List WAL transactions.
     wal.transactions()
 
-    # Flush WAL with garbage collection.
+    # Flush WAL with garbage relation.
```
Comment (kemm) [inline docs/wal.rst]: I suspect that there is still a "collection" 8))

File: tests/conftest.py [hunk 4]
```diff
@@ -193,7 +193,7 @@ def pytest_generate_tests(metafunc):
         tst_conn = tst_db._conn
         bad_conn = bad_db._conn
 
-        if test in {"aql", "collection", "document", "index"}:
+        if test in {"aql", "relation", "document", "index"}:
```
Comment (kemm) [inline tests/conftest.py]: "sql"?

File: dbms/aql.py
Comment (kemm) [inline dbms/aql.py:1]: Maybe it makes sense to rename everywhere `aql` to `sql` at the same time?

**Review by kemm — CHANGES_REQUESTED** [review]

File: docs/aql.rst (hunk 1 above)
Comment (olegrok) [reply docs/aql.rst]: Yes, fixed.

File: docs/wal.rst (hunk 3 above)
Comment (olegrok) [reply docs/wal.rst]: Ooops) Thanks for noticing! Fixed

File: tests/conftest.py (hunk 4 above)
Comment (olegrok) [reply tests/conftest.py]: Fixed

File: dbms/aql.py
Comment (olegrok) [reply dbms/aql.py:1]: Yes. Added a separate commit for that

**Review by kemm — APPROVED** [review]
LGTM

**Pull request merged**